### PR TITLE
Fix: Saved Cart UI styling

### LINKED ISSUE #11438: Saved Cart UI styling update
Update UI for the saved cart listing and details pages to match wire and visual mock ups.

### PR DESCRIPTION
**Pull request by morganm58**
fixes GH-11438

### CODE REVIEW

File: feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html [hunk 1]
```diff
@@ -1,156 +1,154 @@
 <ng-container *ngIf="savedCarts$ | async as savedCarts">
-  <div class="container">
-    <div class="cx-saved-cart-list-header">
-      <h3>
-        {{
-          'savedCartList.savedCarts' | cxTranslate: { count: savedCarts.length }
-        }}
-      </h3>
-    </div>
+  <div class="cx-saved-cart-list-header">
```
Comment (bgambocjaviniar) [inline feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html:2]: As mentioned in the DM,

there's something wrong with the listing page in terms of the description

In epic branch, it's looking like this 
Screen Shot 2021-03-15 at 1 38 07 PM
Screen Shot 2021-03-15 at 1 40 19 PM

whereas, it is now looking like this

Screen Shot 2021-03-15 at 1 36 54 PM
Screen Shot 2021-03-15 at 1 41 08 PM

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]
from comment that I left https://github.com/SAP/spartacus/pull/11504#discussion_r594555974

File: feature-libs/cart/saved-cart/styles/_saved-cart-list.scss [hunk 2]
```diff
@@ -65,29 +79,30 @@ cx-saved-cart-list {
...
 
       .cx-saved-cart-list-value {
         text-decoration: underline;
+        color: var(--cx-color-primary);
       }
     }
 
     .cx-saved-cart-list-make-cart-active {
       .cx-saved-cart-make-active {
         text-transform: uppercase;
```
Comment (developpeurweb) [inline feature-libs/cart/saved-cart/styles/_saved-cart-list.scss]: This seems to be overriding some Desing System standard values, afaik, we only have uppercase text in 3 places: top navigation, table headers and footer headings.
The saved cart list is a table, so it should follow along the same styles as e.g Order History and the same for action links, they should look like the screenshot below:

Screen Shot 2021-03-15 at 3 04 14 PM

Screen Shot 2021-03-15 at 3 28 58 PM

File: feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+  max-height: 70px;
+
   .cx-add-to-saved-cart-container {
     display: flex;
-    justify-content: center;
+    justify-content: start;
     margin-top: 3rem;
 
+    .cx-action-link {
+      font-size: 1rem;
```
Comment (developpeurweb) [inline feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss:10]: These values are overriding the Design System's `.cx-action-link` style. All action links across Spartacus should look the same, as per the modified screenshot:

Screen Shot 2021-03-15 at 3 22 33 PM

**Review by developpeurweb — CHANGES_REQUESTED** [review]

**Review by developpeurweb — APPROVED** [review]
Approving **styles** but Keyboard Accessibility needs to be reviewed and popover repositioning doesn't seem to work as per screenshots, please coordinate with the topic owner for this feature.
Screen Shot 2021-03-15 at 5 42 22 PM
Screen Shot 2021-03-15 at 5 48 52 PM

File: feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss (hunk 3 above)
Comment (bgambocjaviniar) [reply feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss]: > Approving **styles** but Keyboard Accessibility needs to be reviewed and popover repositioning doesn't seem to work as per screenshots, please coordinate with the topic owner for this feature.
> Screen Shot 2021-03-15 at 5 42 22 PM
> Screen Shot 2021-03-15 at 5 48 52 PM

not part of this and it's a known issue. The popover is related to other team's feature which Michal and the team on it are looking to fix

**Comment by bgambocjaviniar** [comment]
> Approving **styles** but Keyboard Accessibility needs to be reviewed and popover repositioning doesn't seem to work as per screenshots, please coordinate with the topic owner for this feature.
> Screen Shot 2021-03-15 at 5 42 22 PM
> Screen Shot 2021-03-15 at 5 48 52 PM

not part of this and it's a known issue. The popover is related to other team's feature which Michal and the team on it are looking to fix

I'm removing the truncate component from the feature for now until it's fully functional in his branch. 

Accessibility-wise (keyboarding) is looking good, but again the truncate (more) component was introduced early and so I removed it

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]
Due to the removal of the component so it doesn't cause any 'noise' for now, can we fix the spacing between description and quantity

Screen Shot 2021-03-16 at 1 17 57 PM



Also, spacing between buttons in the details page 
Screen Shot 2021-03-16 at 1 39 45 PM

### Asked to verify for LTR-RTL as it goes together with styling 
https://github.com/SAP/spartacus/issues/9203 
because can verify and fix accordingly in this PR

**Review by bgambocjaviniar — APPROVED** [review]

**Pull request merged**